src/PLTPlane.cc

```diff
@@ -348,6 +348,15 @@ bool PLTPlane::IsFiducial (FiducialRegion const FidR, int const Col, int const R
       }
       return false;
       break;
+    case kFiducialRegion_FullSensor:
```

Comment:
I believe these warning should be put into a log file.  I do this if I get some free time.